### PR TITLE
Fix the tests so they don't fail with the recent updates

### PR DESCRIPTION
**Pull request by vilyapilya**
https://github.com/logdna/nodejs/pull/76 - changes are good but they break the current tests.
The tests fail bc the code continues to execute after done is called in aftereach block. The initial intention of the block was to clean the logger after each test. I removed afterEach part. It is not needed anymore since each test creates its own logger.

### CODE REVIEW

**Comment by smusali** [comment]
@vilyapilya, please, make sure it passes the linting check

**Review by smusali — CHANGES_REQUESTED** [review]
@vilyapilya, please, make sure it passes the linting check

File: lib/logger.js [hunk 1]
```diff
@@ -384,19 +384,35 @@ const flushAll = function(cb) {
...
+            return cb(`The following errors happened while flushing all loggers: ${errors}`);
+        }
+        return cb();
+
+    };
+
     const errors = [];
     loggers.forEach((logger) => {
         logger._flush((err) => {
             if (err) {
```
Comment (smusali) [inline lib/logger.js:402]: use `error` for readability since the array is `errors` - basically, adding `error` into `errors`: the list of `error`s

**Review by smusali — CHANGES_REQUESTED** [review]

File: lib/logger.js (hunk 1 above)
Comment (vilyapilya) [reply lib/logger.js:402]: I don't think it makes a difference.

**Review by weizou19 — APPROVED** [review]

**Pull request merged**